src/esp32/entry.rs

```diff
@@ -32,15 +34,15 @@ use futures_lite::Future;
 
 pub async fn serve_web_inner(
     app_config: AppClientConfig,
-    tls_server_config: Esp32TlsServerConfig,
+    _tls_server_config: Esp32TlsServerConfig,
     repr: RobotRepresentation,
     _ip: Ipv4Addr,
     webrtc_certificate: WebRtcCertificate,
     exec: Esp32Executor<'_>,
 ) {
     let (mut srv, robot) = {
         let mut client_connector = Esp32Tls::new_client();
-        let mdns = Esp32Mdns::new("".to_string()).unwrap();
+        let mdns = NoMdns {};
```

Comment:
It looks like `Esp32Mdns` has no remaining usage after this change. Should it be removed?

Reply:
no we want to keep it if folks want to use gRPC/HTTP2